src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -139,33 +141,29 @@ export default function RequestedAppointmentDetailsPage() {
       <h1>
         {canceled ? 'Canceled' : 'Pending'} {typeOfCareText} appointment
       </h1>
-      <AlertBox
-        status={canceled ? 'error' : 'info'}
-        className="vads-u-display--block vads-u-margin-bottom--2"
-        backgroundOnly
-      >
-        {canceled && 'This request has been canceled'}
-        {!canceled && (
-          <>
-            Your appointment request has been submitted. We will review your
-            request and contact you to schedule the first available appointment.
-            <div className="vads-u-display--flex vads-u-align-items--center vads-u-color--link-default vads-u-margin-top--2">
-              <i
-                aria-hidden="true"
-                className="fas fa-times vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin-right--1"
-              />
-
-              <button
-                aria-label="Cancel request"
-                className="vaos-appts__cancel-btn va-button-link vads-u-flex--0"
-                onClick={() => dispatch(cancelAppointment(appointment))}
-              >
-                Cancel Request
-              </button>
-            </div>
-          </>
-        )}
-      </AlertBox>
+      {showConfirmMsg && (
+        <AlertBox
+          status={canceled ? 'error' : 'success'}
+          className="vads-u-display--block vads-u-margin-bottom--2"
+          backgroundOnly
+        >
+          {canceled && 'This request has been canceled'}
+          {!canceled && (
+            <>
+              <strong>Your appointment request has been submitted. </strong>
+              We will review your request and contact you to schedule the first
+              available appointment.
+              <br />
+              <div className=" vads-u-margin-top--1">
+                <Link to="/">View your appointments</Link>
+              </div>
+              <div className=" vads-u-margin-top--1">
+                <Link to="/new-appointment">New appointment</Link>
```

Comment:
Were there any analytics or actions associated with these two links on the current confirmation page? We would want to carry them over if so.

Reply:
There isn't for the `« Go back to appointments` button link and the `New appointment` button link is new functionality.

Reply:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/new-appointment/components/ConfirmationPage/index.jsx#L95

Reply:
I think this comment still needs to be addressed. The "View your appointments" and "New appointment" links have events and action being called on the current confirmation page and I think we'd want to do the same for those two links on the confirmation message.

Reply:
Ah I see it now thanks!